### PR TITLE
Edgemos improvments

### LINKED ISSUE #447: No gold in vault of edge
## Description

No gold in the vault of Edge

**Reproduction**

Go edge
Get gold 
No gold in vault

**Screenshots**

image

**Additional context**

S

### PR DESCRIPTION
**Pull request by Haltell**
## About the PR

Fixes #447 

Some minor changes to the atmospherics section on Edge:
- Replaced Liquid Oxygen Canister in the nitrogen chamber with a Nitrogen Canister
- Replaced Combat Gloves with Insulated Gloves
- Replaced input vents with injectors
- Changed plasma pipe colour
- Moved suit storage units into the locker room
- Atmospherics' wardrobe moved to the corner
- Moved rack, steel, and thermomachine board in place of the old suit storage units
- Moved thermomachines so that they are properly oriented, and not on top of pipes
- Linked button to blast doors
- Linked remote signaler to igniter
- Coloured all pipes in atmospherics
- Named most devices in atmospherics
- Set the air mixer to 21% Oxygen, 79% Nitrogen

On top of this a fax machine has been added to the following areas:
- Chief engineer's room
- Engineering front desk

- Glow sticks have been moved into the engineering locker room, and have been replaced with a fax machine

## Why / Balance

- The nitrogen canister replaced the older oxygen canister, as it did not belong.
- Most stations use injectors when inputting into chambers, so it was strange that the previous update to Edge did not.
- The combat gloves in atmospherics was also a strange addition, as they are identical to insulated gloves, but are not related to the engineering department.
- The locker room, and the area between the locker room, and the burn chamber was changed slightly so that it makes more sense.
- The igniter, and button were linked so that it is not required for atmospheric technicians to walk into the chamber to do so.
- The pipes were coloured so that the room is more consistent, even in the inaccessible areas.
- Devices were named, and the mixer was preset to help newer players.
- Fax machines were added because it was strange for the department to go without them.

Overall, nothing has been removed, and only more suitable entities have been put in place of older ones. The only notable addition would be fax machines. All of these should not affect balance at all, and are mostly for the quality of life of engineering, and especially atmospherics players.

## Technical details

This is not a code change.

## Media

- [X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

Only edge.yml has been changed, so there should not be anything relevant here.

**Changelog**

- tweak: Changed Edge Station's Atmospherics
- add: Added Fax Machines to Edge Station's Engineering department

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -3,9 +3,8 @@
 /Content.*/ @DebugOk
 /Content.*/SimpleStation14/ @DEATHB4DEFEAT
 
-/Resources/ @DebugOk @Colin-Tel
-/Resources/ConfigPresets/ @DebugOk
-/Resources/*.yml @DebugOk
+/Resources/*/DeltaV/ @DebugOk
+/Resources/*/Nyanotrasen/ @DebugOk
```
Comment (Colin-Tel) [inline .github/CODEOWNERS]: Why is this changed?

**Review by Colin-Tel — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (Haltell) [reply .github/CODEOWNERS]: Oh, I have no idea, I think it was some issue with me merging from the origin a week or two ago. I personally didn't touch that file, aside from merging what I assumed was the right file to my repository.

**Comment by Colin-Tel** [comment]
Gonna take a look at this today.

**Comment by Colin-Tel** [comment]
image
how the heck does this appear as open in mapping view

**Comment by Colin-Tel** [comment]
Summary of my changes made:
replaced anomalous atmos doors (see above)
renamed `CO2 filter` to `carbon dioxide filter` to keep consistency with other filters
linked atmos's external airlocks to aid against spacing
added a bit of lattice outside the south external airlock so you don't walk directly into space
added diamond plate floor under some airlocks
moved around some storage canisters to free up space in the salvage/atmos shared canister room
extended salvage dock east so you can pilot the shuttle more flush with it
made ai upload have its own power and disconnected the HV power line that runs to it
named all SMESs
adjusted some of the tiles/walls in AME room
adjusted north kitchen layout
fixed #447

**Review by Colin-Tel — DISMISSED** [review]
Good PR, made additional adjustments.

**Pull request merged**